.prettierignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 .next
 src/frontend/.next
-src/frontend/out
\ No newline at end of file
+src/frontend/out
+src/db/testing_data.ts
```

Comment:
prettier throws a syntax error on this file and I have absolutely no idea how to fix it.